versions/models.py

```diff
@@ -557,8 +584,8 @@ def create_versioned_many_to_many_intermediary_model(field, cls, field_name):
         return type(str(name), (Versionable,), {
             'Meta': meta,
             '__module__': cls.__module__,
-            from_: VersionedForeignKey(cls, related_name='%s+' % name),
-            to_field_name: VersionedForeignKey(to, related_name='%s+' % name),
+            from_: VersionedForeignKey(cls, related_name='%s+' % name, auto_created=name),
```

Comment:
Looking through Django's source code, I *think* it's safe.  And the only other alternative that I see would be to add another property to the VersionedForeignKey class (e.g. is_m2m_field).
I'm surprised that we can't tell that it's an intermediary table somehow.